anet-dictionary.yml

```diff
@@ -344,6 +344,7 @@ fields:
       - image/jpeg
       - image/png
       - image/webp
+      - image/svg+xml
```

Comment:
Please keep the list of MIME types in alphabetical order.

Reply:
And don't forget to make the same change to `testDictionaries/no-custom-fields.yml`.